geopandas/tools/sjoin.py

```diff
@@ -43,20 +71,20 @@ def sjoin(
     allowed_hows = ["left", "right", "inner"]
     if how not in allowed_hows:
         raise ValueError(
-            '`how` was "%s" but is expected to be in %s' % (how, allowed_hows)
+            '`how` was "{}" but is expected to be in {}'.format(how, allowed_hows)
         )
 
-    allowed_ops = ["contains", "within", "intersects"]
+    allowed_ops = {"intersects", "contains", "within"}
     if op not in allowed_ops:
         raise ValueError(
-            '`op` was "%s" but is expected to be in %s' % (op, allowed_ops)
+            '`op` was "{}" but is expected to be in {}'.format(op, allowed_ops)
         )
 
     if not _check_crs(left_df, right_df):
         _crs_mismatch_warn(left_df, right_df, stacklevel=3)
```

Comment:
```suggestion
        _crs_mismatch_warn(left_df, right_df, stacklevel=4)
```
I guess this needs to be 4 now. Can you check that?

Reply:
Yes, it should be 4. I changed it to 4 now.